src/devices/GoPiGo3/Models/Motors.cs

```diff
@@ -6,29 +6,32 @@
 
 namespace Iot.Device.GoPiGo3.Models
 {
-
     /// <summary>
-    /// flags -- 8-bits of bit-flags that indicate motor status:
-    /// bit 0 -- LowVoltageFloat - The motors are automatically disabled because the battery voltage is too low
-    /// bit 1 -- Overloaded - The motors aren't close to the target (applies to position control and dps speed control).
+    /// Flags that indicate motor status
     /// </summary> 
     public enum MotorStatusFlags
     {
+        /// <summary>No problems with motors detected</summary>
         AllOk = 0,
+        /// <summary>LowVoltageFloat - The motors are automatically disabled because the battery voltage is too low</summary>
```

Comment:
The name in the summary is not necessary.
```suggestion
        /// <summary>The motors are automatically disabled because the battery voltage is too low.</summary>
```